public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx

```diff
@@ -7,17 +8,22 @@ import { DashboardModel } from '../../state/DashboardModel';
 import { DeleteDashboardButton } from '../DeleteDashboard/DeleteDashboardButton';
 import { TimePickerSettings } from './TimePickerSettings';
 
-interface Props {
+import { updateTimeZoneForSession } from 'app/features/profile/state/reducers';
+import { getTimeSrv } from '../../services/TimeSrv';
+
+interface OwnProps {
   dashboard: DashboardModel;
 }
 
+export type Props = OwnProps & ConnectedProps<typeof connector>;
+
 const GRAPH_TOOLTIP_OPTIONS = [
   { value: 0, label: 'Default' },
   { value: 1, label: 'Shared crosshair' },
   { value: 2, label: 'Shared Tooltip' },
 ];
 
-export const GeneralSettings: React.FC<Props> = ({ dashboard }) => {
+export const GeneralSettingsUnconnected: React.FC<Props> = ({ dashboard, updateTimeZone }) => {
```

Comment:
super nit: we've decided to move away from React.FC<Props> in favor of
```suggestion
export function GeneralSettingsUnconnected({ dashboard, updateTimeZone }:Props): JSX.Element {
```